src/libDirectoryService/DSBlockPreProcessing.cpp

```diff
@@ -407,8 +541,14 @@ bool DirectoryService::VerifyPoWOrdering(
                                    << MAX_MISORDER_NODE << " nodes.");
 
   auto sortedPoWSolns = SortPoWSoln(priorityNodePoWs, true);
-  InjectPoWForDSNode(sortedPoWSolns,
-                     m_pendingDSBlock->GetHeader().GetDSPoWWinners().size());
+
+  // Count the number of actual PoW winners and removed nodes.
+  std::map<PubKey, Peer> dsPoWWinners =
+      m_pendingDSBlock->GetHeader().GetDSPoWWinners();
```

Comment:
Use auto

Reply:
Thanks, changed in https://github.com/Zilliqa/Zilliqa/pull/1587/commits/91acff0ca0f2c1d7140c6c7d3eb05d79ec3f12eb.